src/components/Calculator.js

```diff
@@ -1,46 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleClick = this.handleClick.bind(this);
-  }
+function Calculator() {
```

Comment:
- [ ] OPTIONAL: I recommend using arrow functions as it is today's common practice to use them whenever possible. You may make changes if youfind this comment relevant.